### DISCUSSION TITLE
Diskussion Aufgabe 5)

### DISCUSSION THREAD

**Discussion by JustforFun2112** [JustforFun2112/WAGitHubGruppe11#40] [Show and tell]
Wir fangen an mit der Analyse von Aufg.4)

**Comment by seriadi12** [comment]
Ich habe in Aufgabe 4 mit der Funktion aus 3 a) angefangen, die deskriptive Statistiken für metrische Variablen rechnet. Ich konnte damit das alter analysieren.Dadurch konnte ich einige Grundwerte wichtige Werte wie z.b. Mittelwert oder Median  berechnen.

**Reply by felixhansch** [reply]
Dann erzähl mal was du über das Alter der Studenten rausgefunden hast.

**Reply by seriadi12** [reply]
Das Mindestalter der Studierenden ist 20, das maximale Alter 29. Es gibt also nicht besonders junge oder besonders alte Studenten. Ihr habt also einen realistischen Datensatz, da die meisten Studierenden in diesem Alter sind.

Im Mittel sind die Studierenden ca 25 Jahre alt ( Mean : 24.86). Der Median, also das 50 % Quantil, ist genau 25. Also sind die Studierenden im Durschnitt relativ jung.
Die meisten Studierenden sind 24 Jahre alt (Modus 24)

Als Standardabweichung habe ich 1.874981  bekommen.

**Reply by felixhansch** [reply]
Kannst du daraus einen Schluss im Bezug auf die Verteilung ziehen? Kennst du sie vielleicht?

**Reply by seriadi12** [reply]
Die Verteilung hat ungefähr einen Erwartungswert von 25 und eine Standardabweichung von 2.

**Reply by seriadi12** [reply]
Sie ist symmetrisch, da der Median 25 ist und ungefähr den gleichen Abstand zum Maximum und Minimum hat , nämlich 4.

**Reply by felixhansch** [reply]
25 - 4 = 20 (Minimum) :^)
Ne Spaß beiseite, wir haben die Daten normalverteilt mit den Werten die du oben genannt hast. Anschließend haben wir die Daten gerundet, weil natürlich niemand sein Alter mit Nachkommastellen angibt...
Vielleicht kommt von dem Runden die leichten Abweichungen nach unten.

**Reply by seriadi12** [reply]
Ja der Abstand ist "ungefähr" gleich. Die Wahrscheinlichkeit eines Minimus ist gering

**Reply by seriadi12** [reply]
Also ist die Verteilung warscheinlich normalverteilt.

**Comment by manasii1812** [comment]
Ich begann mit 4 wobei mit Hilfe der Funktion 3 b), das verschiedene geeignete deskriptive Statistiken für kategoriale Variablen berechnet und ausgibt. In diesem Abschnitt habe ich die Spalte "Thema" analysiert.

**Reply by felixhansch** [reply]
Habt ihr da irgendwelche nennenswerten Ergebnisse oder Erkenntnisse erzielen können? Der Datensatz besteht ja nur aus 100 Beobachtungen und ich kann mir eigentlich nicht vorstellen, dass da starke Muster von unseren Einstellungen in Aufgabe 1 geblieben sind.

**Reply by manasii1812** [reply]
wir können feststellen, dass die Stichprobe aus 100 Beobachtungen besteht und keine Werte fehlen. Die Variable nimmt 4 eindeutige Werte an, die die Studienfächer darstellen: Data Science, Informatik, Mathematik und Statistik. Die Stichprobe ist etwas schief in Richtung Date Science und Statistik, wobei weniger Personen Informatik und Mathematik studieren.

**Reply by felixhansch** [reply]
Mit NA's im Datensatz hätten die meisten unserer Funktionen in Aufgabe 3 auch nicht Funktioniert. Glaube selbst sum() hat da ja schon Probleme.
Hast du genauere Erkenntnisse über die Häufigkeiten der Studienfächer und wie würdest du das Interpretieren?

**Reply by manasii1812** [reply]
Aus den Häufigkeitsangaben geht hervor, dass die häufigste Studienrichtung Data Science ist (37 % der Stichprobe), gefolgt von Statistik (30 %), Informatik (23 %) und Mathematik (10 %). Die relativen Häufigkeiten geben uns die gleichen Informationen, ausgedrückt als Anteile. Die Spalten mit den prozentualen und relativen Häufigkeiten liefern dieselben Informationen in unterschiedlichen Formaten.
Insgesamt bieten diese deskriptiven Statistiken eine nützliche Zusammenfassung der "subject"-Variable im Datensatz, die erste Rückschlüsse auf die Verteilung der Studienfächer in der Stichprobe zulässt.

**Reply by felixhansch** [reply]
Also kommt bei euch Data Science > Statistik > Informatik > Mathematik geordnet nach Häufigkeit vor. 
Also wir hatten Data Science und Statistik beiden gleichermaßen die W'keit 3/9 (33,33%) zugeteilt, Informatik 2/9 (22,22%) und Mathematik 1/9 (11,11%).
Mathe und Informatik sind ja bei dem Datensatz fast genau wie nach unseren Einstellungen, bei Data Science und Statistik gabs wohl ein paar Abweichungen...

**Comment by JustforFun2112** [comment]
Ich habe ebenfalls in Aufgabe 4. mitgewirkt und die Funktionen c), e), und f) aus der Aufgabe 3)

wo ich die Korrelation zwischen den einzelnen Variablen und ihrer Quantile bestimmt habe, und diese visuell darstellte.
Wobei die Quantile eine höhere Aussagekraft in visueller Form haben.

**Reply by blanq-dev** [reply]
kannst du mal bitte kurz die die Korrelationen zwischen den Variablen interpretieren?
wie hast die Korrelationen und Quantile in deine Analyse visualisiert ?,falls es nötig war, sie zu visualisieren sonst welche Art von Diagrammen oder Grafiken am besten geeignet waren, um diese Informationen darzustellen ?
zusätzlich möchten wir erfahren, ob die Quantilen dazu beigetragen haben, Ausreißer in den Daten zu reduzieren, und die Verteilungen den Daten gut zu visualisieren.

**Reply by JustforFun2112** [reply]
Korrelationen Entstanden nur minimal Zwischen dem Interesse an Mathematik  und  der Programmierung im Zusammenhand mit der Wahl eines Mathe Lk's.

dafür war eine rein Werte bezogene Analyse, ähnlich wie wir es in den Modulen Statistik gelernt haben. Es war keine Visualisierung  notwendig.

dadurch wurde  kein Offensichtlicher Zusammenhang klar.

Aber man kann erkennen das die Probanden, die einen LK besuchten, ein geringeres Interesse Interesse and Mathe und Programmierung hatten. Aber zwischen den Interessen selber gab es keine eindeutige Korrelation



Zu den Quantilen gibt das eine übersichtlichere Angabe wie die Unterteilung aussieht.
Dabei ist aber eine Grafik wichtig um bei den 100 Testungen auch einen Überblick zu haben.

mit 3 Balken um die Interpretation "niedrig" "mittel" "hoch".
Es wird sichtbar dass hauptsächlich jüngere Studierende vorhanden sind,
im Allgemeinen ein durchschnittliches Interesse and Mathematik und Programmierung überwiegt.
Und die Programmierung doch von mehr Studierenden als Interessant gedeutet wurde als Mathematik



deutlich wird es and er Aufteilung da 82% aller Befragten Studenten den Mathe-LK belegt habe. Um aber weitere Deutungen zu erheben, müsste man einen Studiengang nehmen, der nicht mathematisch angelehnt ist, z.B. Sprachwissenschaften oder Kunst und Musik.

**Reply by blanq-dev** [reply]
hat die LK Daten duch den Graph auch was über die anderen Studiengänge bzw. diejenige, die Data Science und Statistik studieren , vermittelt?

**Reply by JustforFun2112** [reply]
nur das ca. 20% von allen Studiengängen keinen LK gewählt haben

**Comment by manasii1812** [comment]
In der Aufgabe 4 wird die Funktion aus 3 d) verwendet, wobei die Funktion bivariate_stats einen Datenrahmen (df), eine metrische Spalte (metric_col) und eine dichotome Spalte (dich_col) als Eingaben annimmt. Sie filtert dann den Datenrahmen, um nur Beobachtungen mit nicht fehlenden Werten in beiden Spalten einzuschließen, berechnet den Mittelwert der metrischen Variable für jedes Niveau der dichotomen Variable und berechnet Cohen's d, um die Effektgröße des Unterschieds der Mittelwerte zwischen den beiden Gruppen zu quantifizieren.

**Reply by felixhansch** [reply]
Das heißt du hast den Zusammenhang von Alter als metrische Variable und Mathe LK Ja oder Nein als dichotome Variable untersucht?
Was hat sich dabei so ergeben?

**Reply by manasii1812** [reply]
Die Ergebnisse zeigen, dass das Durchschnittsalter der Schüler, die Mathe LK belegt haben, etwas niedriger ist (24,8) als das der Schüler, die keinen Mathe LK belegt haben (25,1). Dies deutet darauf hin, dass Schüler, die Mathe LK belegen, tendenziell etwas jünger sind. Außerdem beträgt die Effektgröße (Cohen's d) 0,127, was auf einen kleinen Effekt hinweist. Insgesamt deutet das Ergebnis darauf hin, dass es einen signifikanten, aber geringen Zusammenhang zwischen Alter und Mathe LK gibt.

**Reply by felixhansch** [reply]
Aber spricht ein Effekt von 0,127 wirklich für einen nennenswerten Effekt? Ist ja schon fast nahe der 0 ist. Bedenke, dass wir nur eine Stichprobengröße von 100 haben.

**Reply by seriadi12** [reply]
Ich denke schon, da der Wertebereich ja sowieso nur zwischen 0 und 1 liegt oder ?

**Reply by felixhansch** [reply]
Glaube Cohen's d kann Werte von -Inf bis Inf annehmen. Ist aber unwahrscheinlich.
Ich würde es ähnlich zum Korrelationskoeffizienten interpretieren.

**Reply by manasii1812** [reply]
Eine Effektgröße von 0,127 mag zwar auf einen kleinen Effekt hindeuten, aber es ist wichtig, auch die Stichprobengröße und die statistische Signifikanz zu berücksichtigen. Bei einer Stichprobengröße von 100 können selbst kleine Effekte statistisch signifikant sein. 
Um die statistische Signifikanz der Effektgröße zu bestimmen, müssten den p-Wert berechnen. Ohne diese Information ist es schwierig zu sagen, ob der Effekt signifikant ist oder nicht.

**Reply by blanq-dev** [reply]
kannst du vielleicht aus deiner Sicht die von dir über die Daten folgende Schlussfolgerung " Dies deutet darauf hin, dass Schüler, die Mathe LK belegen, tendenziell etwas jünger sind" interpretieren.

**Reply by manasii1812** [reply]
Anhand des Ergebnisses der bivariaten Analyse können wir feststellen, dass es einen kleinen, aber statistisch signifikanten Unterschied im Durchschnittsalter der Schüler gibt, die Mathe-LK belegen, und derjenigen, die dies nicht tun. Es gibt einen kleinen Effekt, aber dieser Effekt ist angesichts der Stichprobengröße von 100 immer noch statistisch signifikant. Es ist jedoch zu beachten, dass die Effektgröße zwar statistisch signifikant ist, der tatsächliche Altersunterschied zwischen den beiden Gruppen jedoch relativ gering ist.

**Reply by manasii1812** [reply]
wir sollten vorsichtig sein, wenn es darum geht, eindeutige Schlussfolgerungen über die praktische Bedeutung dieses Effekts zu ziehen, ohne andere Faktoren zu berücksichtigen, die die Beziehung beeinflussen könnten

**Reply by felixhansch** [reply]
> Eine Effektgröße von 0,127 mag zwar auf einen kleinen Effekt hindeuten, aber es ist wichtig, auch die Stichprobengröße und die statistische Signifikanz zu berücksichtigen. Bei einer Stichprobengröße von 100 können selbst kleine Effekte statistisch signifikant sein. Um die statistische Signifikanz der Effektgröße zu bestimmen, müssten den p-Wert berechnen. Ohne diese Information ist es schwierig zu sagen, ob der Effekt signifikant ist oder nicht.

Ist es nicht andersherum? Bei einer Stichprobengröße von "nur" 100 würde ich einen so schwachen Effekt nicht all zu ernst nehmen. Man bräuchte weitaus mehr Daten oder nicht?

**Comment by blanq-dev** [comment]
hat jemand die Mittelwerte, Median und Standardabweichungen von den Daten im Bezug auf  Interesse an 
Mathematik und die Interesse an Programmieren berechnet und kann uns vielleicht mitteilen,
welche Schlüsse daraus gezogen werden kann.

**Reply by JustforFun2112** [reply]
Dazu habe ich keine Analyse erstellen können, und bin dementsprechend nicht darauf gekommen ob es zusammenhänge dazwischen gibt.
